platform/linux-generic/odp_schedule_if.c

```diff
@@ -25,11 +25,57 @@ extern const schedule_api_t schedule_scalable_api;
 const schedule_fn_t *sched_fn;
 const schedule_api_t *sched_api;
 
+#ifdef ODP_DEBUG
+int _odp_schedule_configured;
+#endif
+
 uint64_t odp_schedule_wait_time(uint64_t ns)
 {
 	return sched_api->schedule_wait_time(ns);
 }
 
+int odp_schedule_capability(odp_schedule_capability_t *capa)
+{
+	return sched_api->schedule_capability(capa);
+}
+
+#ifdef ODP_DEBUG
+int odp_schedule_config(const odp_schedule_config_t *config)
```

Comment:
ODP_DEBUG is always defined. Also it's better not to duplicate functions, but use the define value so that optimizer removes extra debug code:

if (ODP_DEBUG == 1 &&  _odp_schedule_configured)
		ODP_ABORT("Scheduler already configured!\n");

Actually, in this case ODP_ASSERT() would do the job (prints error and aborts).

Reply:
Ack